### PR TITLE
6883 update msp direct actions

### PR DESCRIPTION
**Pull request by daveb-hni**
Updates to the existing MSP Direct actions including:

- Adding authorize functionality to deploy action
- Standardizing tags
- Adding env suffix to images e.g. -dev for mspdirect-frontend-dev
- Updating comment in gitops commit to provide more detail with version, env and action specified.

### CODE REVIEW

File: .github/workflows/DevEnv-frontend-BuildImageAndPush.yml [hunk 1]
```diff
@@ -7,111 +7,96 @@ env:
...
-    # Determine the version number tag i.e. Release_1.0_<branch-name>_<build execution> 
-    - name: Determine Version number
-      # Branch name is fixed to 'main' in version identifier
-      run: |
-        echo "RELEASE_VERSION=Dev_Build_${{ github.event.inputs.branch }}_${{github.run_number}}" | tee -a $GITHUB_ENV
+      # Determine the version number tag i.e. Release_1.0_<branch-name>_<build execution>
+      - name: Determine Version number
+        # Branch name is taken from action inputs
+        run: |
+          echo "VERSION=Dev_Build_${{ github.event.inputs.branch }}_${{github.run_number}}" | tee -a $GITHUB_ENV
```
Comment (pankaj-cgi) [inline .github/workflows/DevEnv-frontend-BuildImageAndPush.yml:40]: to lower case?

File: .github/workflows/DevEnv-frontend-BuildImageAndPush.yml [hunk 2]
```diff
@@ -7,111 +7,96 @@ env:
...
-      uses: redhat-actions/buildah-build@v2
-      with:
-        image: ${{ env.IMAGE_NAME }}
-        tags: ${{ env.IMAGE_TAGS }}
-        context: ./frontend
-        dockerfiles: |
-          ./frontend/Dockerfile
+      # Determine the tags
+      - name: Determine image tags
+        if: env.IMAGE_TAGS == ''
```
Comment (pankaj-cgi) [inline .github/workflows/DevEnv-frontend-BuildImageAndPush.yml:50]: we dont need this if check any more. Same for all the actions

**Review by pankaj-cgi — CHANGES_REQUESTED** [review]

File: .github/workflows/DevEnv-frontend-BuildImageAndPush.yml (hunk 2 above)
Comment (daveb-hni) [reply .github/workflows/DevEnv-frontend-BuildImageAndPush.yml]: Removed all these unnecessary checks

File: .github/workflows/DevEnv-frontend-BuildImageAndPush.yml (hunk 1 above)
Comment (daveb-hni) [reply .github/workflows/DevEnv-frontend-BuildImageAndPush.yml]: updated to lowercase

**Review by pankaj-cgi — APPROVED** [review]
Changes look good.

**Pull request merged**